qrisk/stats.py

```diff
@@ -238,37 +235,43 @@ def annual_volatility(returns, period=DAILY):
         return np.nan
 
     try:
-        ann_factor = ANNUALIZATION_FACTORS[period]
+        ann_factor = annualization[period]
     except KeyError:
         raise ValueError(
-            "period cannot be: '{}'."
-            " Must be '{}', '{}', or '{}'".format(
-                period, DAILY, WEEKLY, MONTHLY
+            "period cannot be '{}'. "
+            "Can be '{}'.".format(
+                period, "', '".join(annualization.keys())
             )
         )
 
-    return returns.std() * np.sqrt(ann_factor)
+    volatility = returns.std() * pow(ann_factor, float(1.0/alpha))
```

Comment:
When calculating volatility, it is sometimes useful to use slightly different values here. There might be some users that want the flexibility.

https://en.wikipedia.org/wiki/Volatility_(finance)#Mathematical_definition